src/EventProcessor/OrderEventProcessor.php

```diff
@@ -23,12 +21,12 @@ public function __construct(private RobinClient $robinClient)
     }
 
     /**
-     * @param EventInterface|OrderEvent $event
```

Comment:
Same here.

Reply:
Same, reverted reference to concrete event class.